spec/diagnose_spec.rb

```diff
@@ -25,24 +25,16 @@
       case @runner.type
       when :ruby
         {
-          "gem_version" => VERSION_PATTERN,
-          "name" => "DiagnoseTests"
+          "gem_version" => VERSION_PATTERN
         }
-      when :elixir
-        {
-          "name" => "DiagnoseTests"
-        }
-      when :nodejs
-        {
-          "name" => be_empty.or(be_nil)
-        }
-      end
+      else {} end
```

Comment:
```suggestion
      end
```

The merge call already falls back on an empty hash.